plugins/go-build/bin/go-build

```diff
@@ -760,7 +760,13 @@ unset IPV6
 
 GO_BUILD_INSTALL_PREFIX="$(abs_dirname "$0")/.."
 
-IFS=: GO_BUILD_DEFINITIONS=($GO_BUILD_DEFINITIONS ${GO_BUILD_ROOT:-$GO_BUILD_INSTALL_PREFIX}/share/go-build)
+if [[ $USE_FAKE_DEFINITIONS == "true" ]]; then
+  DIR_SUFFIX=test/fixtures/definitions
+else
+  DIR_SUFFIX=share/go-build
+fi
```

Comment:
Can you add a comment on top of this to explain what does "using fake definitions" do? From what I gather, it's useful for testing, to "mock" the available go definitions.

Reply:
Done